vocab/properties.json

```diff
@@ -595,22 +569,22 @@
         ]
     },
     "email": {
-        "description": null,
+        "description": "Address to which an electronic mail can be sent.",
         "name": "Email",
         "sameAs": null,
         "schemas": [
             "core/schemas/v3/actors/contactInformation.schema.tpl.json"
         ]
     },
     "endDate": {
-        "description": null,
+        "description": "Date at which something terminates in time",
```

Comment:
I leave it as it is. It's not necessary to further define it I think. If we want to be precise it would be "Date in the Gregorian calendar at which..." But I think that's over the top here.